charts/sn-platform-slim/values.yaml

```diff
@@ -1168,16 +1170,19 @@ broker:
     extraOptions: []
     gcLoggingOptions: []
   configData:
-    # @deprecated:
-    # 1. You can use a service account to assume an IAM role to access S3 bucket
-    # 2. You can use a k8s secret to reference AWS credentials. See `offload.s3.secret`
-    # AWS_ACCESS_KEY_ID: "[YOUR AWS ACCESS KEY ID]"
-    # AWS_SECRET_ACCESS_KEY: "[YOUR SECRET]"
     managedLedgerDefaultEnsembleSize: "3"
     managedLedgerDefaultWriteQuorum: "3"
     managedLedgerDefaultAckQuorum: "2"
     PULSAR_PREFIX_systemTopicEnabled: "true"
     PULSAR_PREFIX_topicLevelPoliciesEnabled: "true"
+    # Enable the new delayed message
+    delayedDeliveryTrackerFactoryClassName: "org.apache.pulsar.broker.delayed.BucketDelayedDeliveryTrackerFactory"
```

Comment:
Well, I think maybe we should support dynamic calculate of the segment size or bucket size by cluster resources in the future. :) 

/cc @coderzc WDYT?

Reply:
Yes, but we need to design a set of calculation formulas.